lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -61,7 +64,8 @@ internal sealed class REEFIMRUClient<TMapInput, TMapOutput, TResult> : IIMRUClie
         IEnumerable<TResult> IIMRUClient<TMapInput, TMapOutput, TResult>.Submit(IMRUJobDefinition jobDefinition)
         {
             string driverId = string.Format("IMRU-{0}-Driver", jobDefinition.JobName);
-
+            var overallPerMapConfig = Configurations.Merge(jobDefinition.PerMapConfigGeneratorConfig.ToArray());
```

Comment:
You might want to put this into a `try/catch` and log the errors in case of merge conflicts. That way, the log contains information about why it failed.